website/docs/r/route.html.markdown

```diff
@@ -89,3 +89,29 @@ will be exported as an attribute once the resource is created.
 
 - `create` - (Default `2 minutes`) Used for route creation
 - `delete` - (Default `5 minutes`) Used for route deletion
+
+## Import
+
+Individual routes can be imported using a pseudo ID defined as r-ROUTETABLEID_DESTINATION.
```

Comment:
If we simplify the using something like the example import function mentioned earlier, we can remove `r-` from import. 👍